<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PUBGIS_GUI</class>
 <widget class="QMainWindow" name="PUBGIS_GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>769</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PUBGIS_GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="map_creation_view">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progress_bar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="video_tab">
       <attribute name="title">
        <string>Video</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="2" column="3">
            <widget class="QTimeEdit" name="landing_time">
             <property name="displayFormat">
              <string>mm:ss</string>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <widget class="QTimeEdit" name="death_time">
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::UpDownArrows</enum>
             </property>
             <property name="currentSection">
              <enum>QDateTimeEdit::MinuteSection</enum>
             </property>
             <property name="displayFormat">
              <string>mm:ss</string>
             </property>
            </widget>
           </item>
           <item row="1" column="7">
            <widget class="QPushButton" name="video_file_browse_button">
             <property name="text">
              <string>Browse</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3" colspan="4">
            <widget class="QLineEdit" name="video_file_edit"/>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="image_label_2">
             <property name="text">
              <string>Landing Time</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QLabel" name="image_label_3">
             <property name="text">
              <string>Death Time</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="video_file_label">
             <property name="text">
              <string>Video</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="live_tab">
       <attribute name="title">
        <string>Live</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <item row="0" column="3">
        <widget class="QPushButton" name="output_file_browse_button">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="output_file_edit"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="output_file_label">
         <property name="text">
          <string>Output File</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="2">
        <widget class="QLabel" name="path_color_label">
         <property name="text">
          <string>Path Color</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="8">
        <widget class="QPushButton" name="cancel_button">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QPushButton" name="color_select_button">
         <property name="text">
          <string>Change Color</string>
         </property>
        </widget>
       </item>
       <item row="3" column="7">
        <widget class="QPushButton" name="process_button">
         <property name="text">
          <string>Process</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="path_color_preview">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="time_step">
         <item>
          <property name="text">
           <string>0.25</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="image_label_4">
         <property name="text">
          <string>Time Step</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
